vms/platformvm/vm_regression_test.go

```diff
@@ -2129,12 +2130,18 @@ func checkValidatorBlsKeyIsSet(
 	}
 
 	val, found := vals[nodeID]
-	if !found {
+	switch {
```

Comment:
This was only changed because the prior code required the usage of caching in the underlying state package to pass (which was definitely unintended)